mne/channels/channels.py

```diff
@@ -502,8 +502,9 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         title : str | None
             Title for the figure. If None (default), equals to ``'Sensor
             positions (%s)' % ch_type``.
-        show_names : bool
-            Whether to display all channel names. Defaults to False.
+        show_names : bool | array of str
```

Comment:
IMO the API at ``plot_topomap`` is quite crappy. If we unify with ``plot_topomap``, it would make this less usable. For one, if you pass ``show_names=True``, you would have to set ``names=[...]`` to get anything shown since ``names=None`` plots nothing.